sympy/galgebra/ga.py

```diff
@@ -23,7 +23,7 @@
 
 from __future__ import print_function
 
-from functools import reduce
+from sympy.core.compatiblity import reduce
```

Comment:
You spelled `compatibility` wrong here, which caused most of the tests to fail. Please run the tests with `.bin/test` before pushing, as they will easily catch stuff like this. Read this: https://github.com/sympy/sympy/wiki/Development-workflow